seaborn/_base.py

```diff
@@ -933,7 +933,8 @@ def iter_data(
 
             grouping_keys = []
             for var in grouping_vars:
-                grouping_keys.append(levels.get(var, []))
+                key = levels.get(var) if levels.get(var) is not None else []
+                grouping_keys.append(key)
```

Comment:
Total nitpick, but the duplicate call to `levels.get` above looks cumbersome, maybe move the ternary down here? e.g.

```suggestion
                key = levels.get(var)
                grouping_keys.append([] if key is None else key)
```